src/Widgets/calendar/GridDay.vala

```diff
@@ -115,6 +115,8 @@ public class DateTime.Widgets.GridDay : Gtk.EventBox {
                         event_dots[event_uid] = event_dot;
 
                         event_grid.add (event_dot);
+                        grid.attach (event_grid, 0, 1);
+                        grid.margin_top = 6;
```

Comment:
Put this at grid creation time, not here

Reply:
Done.